packages/control-hub/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@rev-robotics/control-hub",
+  "version": "0.1.0",
+  "description": "High level library for the REV Robotics Control Hub",
+  "main": "dist/index.js",
+  "type": "module",
+  "dependencies": {
+    "isomorphic-ws": "^5.0.0",
+    "axios": "^1.4.0",
+    "semver": "^7.5.1",
+    "@rev-robotics/rev-hub-core": "*"
+  },
+  "peerDependencies": {
+    "get-port": "6.x"
```

Comment:
get-port is no longer used by this package, right? We just want to document how to use it with adbkit to set up port forwarding.